composer.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "novaway/filemanagementbundle",
     "type": "symfony-bundle",
-    "description": "Bundle for File and Image management in Symfony 2",
+    "description": "Bundle for File and Image management in Symfony 3",
```

Comment:
Don't mention the framework version. The bundle is not specific to a version, but to the framework.